frontend/demo/component/avatar/avatar-group-max-items.ts

```diff
@@ -0,0 +1,31 @@
+import '../../init'; // hidden-full-source-line
+
+import { html, LitElement, customElement } from 'lit-element';
+import '@vaadin/vaadin-avatar/vaadin-avatar-group';
+import { applyTheme } from 'themes/theme-generated.js';
+import { AvatarGroupItem } from '@vaadin/vaadin-avatar/src/interfaces';
+
+@customElement('avatar-group-basic')
```

Comment:
Conflicting custom element name
```suggestion
@customElement('avatar-group-max-items')
```

Reply:
Done.